tests/Tests/ORM/Functional/Ticket/SwitchContextWithFilter/User.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket\SwitchContextWithFilter;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ */
+class User
```

Comment:
`User` is a reserved keyword in PostgreSQL. You should probably use a `Table` annotation here as well because of that.